src/HtmlUtils.js

```diff
@@ -102,10 +102,10 @@ var sanitizeHtmlParams = {
     // URL schemes we permit
     allowedSchemes: ['http', 'https', 'ftp', 'mailto'],
 
-    // DO NOT USE. sanitize-html allows all URL starting with '//'
-    // so this will always allow links to whatever scheme the
-    // host page is served over.
-    allowedSchemesByTag: {},
+    allowedSchemesByTag: {
+        img: [ 'data', 'mxc' ],
```

Comment:
My earlier proposal for this was to use a data attribute, like `data-src` to make what's going on clearer. This would have the added benefit of providing a fallback in the real src attribute.